### PR TITLE
Homework 6

### PR DESCRIPTION
**Pull request by Syreon514**

### CODE REVIEW

File: homeworks/alexander.yatsenko_Syreon514/homework_6/task1.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+function createUser(user) {
+    let value;
```
Comment (markoffden) [inline homeworks/alexander.yatsenko_Syreon514/homework_6/task1.js]: using single protected variable for both assignments, not good

**Comment by markoffden** [comment]
in general OK, but please fix first task as per comment

File: homeworks/alexander.yatsenko_Syreon514/homework_6/task1.js (hunk 1 above)
Comment (Syreon514) [reply homeworks/alexander.yatsenko_Syreon514/homework_6/task1.js]: Understood. Fixed

**Comment by Syreon514** [comment]
Task 1 fixed

**Pull request merged**